example/src/test_utils.ts

```diff
@@ -0,0 +1,43 @@
+import { EncodedContent } from "../../src/XMTP.types";
+import { CodecError } from "../../src/lib/CodecError";
+import { ContentTypeID } from "../../src/lib/ContentTypeID";
+import { Compression } from "../../src/XMTP.enums";
+
+export class NumberCodec {
+  contentType: {
+    id(): string;
+    authorityID: string;
+    typeID: string;
+    versionMajor: number;
+    versionMinor: number;
+  };
+
+  constructor() {
+    this.contentType = new ContentTypeID({
+      authorityID: "example.com",
+      typeID: "number",
+      versionMajor: 1,
+      versionMinor: 1,
+    });
+  }
+
+  encode<T>(content: T) {
+    const encodedContent = {
+      type: this.contentType,
+      content: JSON.stringify(content),
+      compression: Compression.COMPRESSION_DEFLATE,
```

Comment:
I think we might want to leave this out because we're not actually doing any compression at the moment.

Reply:
Alright, logic here was to add in what we're going to need to consume in the next few weeks, but will remove so we don't introduce unused code.